resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DbReader.scala

```diff
@@ -27,13 +27,16 @@ object DbReader {
 
   val deletedDisksQuery =
     sql"""
-           select pd1.id, pd1.googleProject, pd1.name from PERSISTENT_DISK as pd1 where pd1.status="Deleted" and
-           NOT EXISTS
-           (
-             SELECT *
-             FROM PERSISTENT_DISK pd2
-             WHERE pd1.googleProject = pd2.googleProject and pd1.name = pd2.name and pd2.status != "Deleted"
-          )
+           select pd1.id, pd1.googleProject, pd1.name 
+           FROM PERSISTENT_DISK as pd1 
+           WHERE pd1.status="Deleted" and
```

Comment:
small nit, but typically its good to put the linking conditional at the beginning of the next line in long queries, I think it read a bit better (i.e. move the `AND` thats above + below to the beginning of the subsequent line

Reply:
Ha..interesting..I have the opposite preference (i.e. having `AND` at the end like it is above). I understand that it's a personal taste and wouldn't push back against. The one thing I do feel a bit strongly about is consistency between queries, though. Would love to pick one style and apply it to all if possible 😊

Reply:
I like having `AND` at the end as well...going to leave things as is since it's just personal preference